app/assets/stylesheets/cypress/_buttons.scss

```diff
@@ -41,15 +43,15 @@ td a.cmd, td input.cmd {
   font-size:85%;
 }
 
+// delete button hover color
 a.del:hover {
-background-color: $dltHoverBkg;
+background-color: #ccc; // gray
```

Comment:
Similarly, we can probably just change the value of `dltHoverBkg`, or create one variable for both (something like `$hoverBkg`?) These variables are declared in `app/assets/stylesheets/cypress/_variables.scss`.